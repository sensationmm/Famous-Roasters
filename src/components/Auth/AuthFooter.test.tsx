import { MockedProvider } from '@apollo/client/testing'
import { render, waitFor } from '@testing-library/react'
import React from 'react'
import { MemoryRouter } from 'react-router-dom'

import { AuthFooter } from '.'

describe('Auth footer', () => {
  const snippet = () => (
    <MemoryRouter initialEntries={['/login']}>
      <AuthFooter />
    </MemoryRouter>
  )

  it('Renders correctly', async () => {
    const { container } = render(
      <MockedProvider defaultOptions={{ watchQuery: { fetchPolicy: 'no-cache' } }}>{snippet()}</MockedProvider>,
    )
    await waitFor(() => new Promise((res) => setTimeout(res, 0)))
    expect(container).toMatchSnapshot()
  })
})
