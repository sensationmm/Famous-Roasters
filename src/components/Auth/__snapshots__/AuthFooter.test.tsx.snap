// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth footer Renders correctly 1`] = `
<div>
  <div
    class="flex"
  >
    <div
      class="border-t border-brand-grey-whisper absolute w-full left-0"
    />
  </div>
  <div
    class="w-full mt-4 flex justify-center"
  >
    <a
      href="//www.notion.so/Datenschutzhinweise-ad5b45261a964166a74883d2770e28ea"
      target="_blank"
    >
      <div
        class="font-normal border-b text-xs leading-4"
      >
        Datenschutz-Bestimmungen
      </div>
    </a>
    <a
      class="ml-10"
      href="//slender-petunia-c3c.notion.site/Imprint-33b18fd9e469471ab768eef8155da968"
      target="_blank"
    >
      <div
        class="font-normal border-b text-xs leading-4"
      >
        Impressum
      </div>
    </a>
  </div>
</div>
`;
